packages/dara-core/js/api/http.tsx

```diff
@@ -48,6 +49,19 @@ export class RequestExtrasSerializable {
             serializable.options.headers = Object.fromEntries(headers.entries());
         }
 
+        return serializable;
+    }
+
+    /**
+     * Serialize the extras into a string.
+     */
+    toJSON(): string | null {
+        const serializable = this.toSerializable();
+
+        if (!serializable) {
```

Comment:
Do we need any warnings if it is not serializable? Not sure when this would happen but wondering if it might make it easier to pinpoint issues

Reply:
This just exists in case `toSerializable` returns null/undefined, not if the value is not serializable